app/kafka/manager/ClusterManagerActor.scala

```diff
@@ -90,6 +90,9 @@ class ClusterManagerActor(cmConfig: ClusterManagerActorConfig)
   private[this] val ksProps = Props(classOf[KafkaStateActor],sharedClusterCurator, adminUtils.isDeleteSupported, cmConfig.clusterConfig)
   private[this] val kafkaStateActor : ActorPath = context.actorOf(ksProps.withDispatcher(cmConfig.pinnedDispatcherName),"kafka-state").path
 
+  private[this] val lksProps = Props(classOf[LogkafkaStateActor],sharedClusterCurator, adminUtils.isDeleteSupported, cmConfig.clusterConfig)
```

Comment:
We only want to enable the log kafka specific functionality if it is enabled in the ClusterConfig.  So that means all of these should be optional (e.g. Option[...])